examples/extensions/add_google_my_business_location_extensions.rb

```diff
@@ -0,0 +1,209 @@
+#!/usr/bin/env ruby
+# Encoding: utf-8
+#
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# This example adds a feed that syncs feed items from a Google My Business (GMB)
+# account and associates the feed with a customer.
+
+require 'optparse'
+require 'google/ads/google_ads'
+require 'date'
+
+def add_google_my_business_location_extensions(
+  customer_id,
+  gmb_email_address,
+  gmb_access_token,
+  business_account_identifier)
+  # GoogleAdsClient will read a config file from
+  # ENV['HOME']/google_ads_config.rb when called without parameters
+  client = Google::Ads::GoogleAds::GoogleAdsClient.new
+
+  gmb_feed_resource_name = create_feed(
+    client,
+    customer_id,
+    gmb_email_address,
+    gmb_access_token,
+    business_account_identifier,
+  )
+
+  create_customer_feed(client, customer_id, gmb_feed_resource_name)
+end
+
+def create_feed(
+  client,
+  customer_id,
+  gmb_email_address,
+  gmb_access_token,
+  business_account_identifier)
+  # Creates a feed operation.
+  operation = client.operation.create_resource.feed do |feed|
+    feed.name = "Google My Business feed #{(Time.new.to_f * 1000).to_i}"
+    feed.origin = :GOOGLE
+    feed.places_location_feed_data = client.resource.places_location_feed_data do |data|
+      data.email_address = gmb_email_address
+      data.business_account_id = business_account_identifier
+      data.label_filters << "Stores in New York"
+      data.oauth_info = client.resource.o_auth_info do |oauth|
+        oauth.http_method = "GET"
+        oauth.http_request_url = "https://www.googleapis.com/auth/adwords"
+        oauth.http_authorization_header = "Bearer #{gmb_access_token}"
```

Comment:
I think we need to scrub this field in our logging implementation. Can you check with Anash?

Reply:
@AnashOommen 
Hi Anash - Want to confirm with you, for this `oauth.http_authorization_header` field, we need to scrub it in our logging, right?

Reply:
It looks like a candidate for scrubbing, but IIRC we don't do it today. Could you add a frontlog entry and possibly a followup CL to add scrubbing?

Reply:
Sure. I've added to our client lib front log. Will revisit and add a followup CL.